Kernel/Devices/Storage/StorageManagement.cpp

```diff
@@ -205,21 +211,26 @@ UNMAP_AFTER_INIT ErrorOr<NonnullOwnPtr<Partition::PartitionTable>> StorageManage
     return TRY(Partition::GUIDPartitionTable::try_to_initialize(device));
 }
 
+void StorageManagement::enumerate_device_partitions(StorageDevice& device)
+{
+    auto partition_table_or_error = try_to_initialize_partition_table(device);
+    if (partition_table_or_error.is_error())
+        return;
```

Comment:
Maybe make this return `ErrorOr<void>` now it's a separate function?

Reply:
Sure, but we are just swallowing the error for now